polkadot/roadmap/implementers-guide/src/runtime/parainherent.md

```diff
@@ -60,3 +60,34 @@ processing it, so the processed inherent data is simply dropped.
 This also means that the `enter` function keeps data around for no good reason. This seems acceptable though as the size
 of a block is rather limited. Nevertheless if we ever wanted to optimize this we can easily implement an inherent
 collector that has two implementations, where one clones and stores the data and the other just passes it on.
+
+## Data sanitization
```

Comment:
Data sanitization has a predefined meaning, which is different: https://en.wikipedia.org/wiki/Data_sanitization
```suggestion
## Sanitization

```